index.d.ts

```diff
@@ -119,6 +119,12 @@ declare module "@luxuryescapes/router" {
      * @defaultValue `{}`
      */
     jsonOptions?: { [option: string]: string | number | boolean | null | undefined };
+    
+    /**
+     * Cache-Control header value
+     * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control
+     */
+    cacheControlString?: string;
```

Comment:
Would it be better to abstract away the syntax and make it easier to use, e.g.

```suggestion
cacheControl?: {
  type: 'public', // More options can be added when required
  TTL: number,
  }
```

Reply:
I don't think we should, this would limit it too much, the spec is very wide and has many options that can be combined in different ways.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control